seirmo/_models.py

```diff
@@ -5,29 +5,60 @@
 #
 
 import numpy as np
+import pints
 from scipy.integrate import solve_ivp
 
 
-class ForwardModel(object):
-    """ForwardModel Class:
-    Abstract base class for any models.
+class ForwardModel(pints.ForwardModel):
+    """
+    Abstract base class for forward models.
+    Extends :class:`pints.ForwardModel`.
     """
-
     def __init__(self):
         super(ForwardModel, self).__init__()
 
+    def n_outputs(self):
+        """
+        Returns the number of model outputs.
+        """
+        raise NotImplementedError
+
+    def n_parameters(self):
+        """
+        Returns the number of model parameters.
+        """
+        raise NotImplementedError
+
+    def output_names(self):
+        """
+        Returns the names of the model outputs.
+        """
+        raise NotImplementedError
+
+    def parameter_names(self):
+        """
+        Returns the names of the model parameters.
+        """
+        raise NotImplementedError
+
+    def set_outputs(self, outputs):
+        """
+        Sets the outputs of the model.
+        """
+        raise NotImplementedError
+
     def simulate(self, parameters, times):
         """
         Forward simulation of a model for a given time period
         with given parameters
         Returns a sequence of length ``n_times`` (for single output problems)
         or a NumPy array of shape ``(n_times, n_outputs)`` (for multi-output
         problems), representing the values of the model at the given ``times``.
-
-        Parameters
-        ----------
-        parameters: sequence of numerics
-        times: sequence of numerics
+        :param parameters: An array-like object with parameter values
```

Comment:
It's very possible that I did this, but could you insert a blank line here, i.e.
```
or a NumPy array of shape ``(n_times, n_outputs)`` (for multi-output
problems), representing the values of the model at the given ``times``.	       
problems), representing the values of the model at the given ``times``.

:param parameters: An array-like object with parameter values
     of length :meth:`n_parameters`.
```